src/oge/data_cleaning.py

```diff
@@ -2145,3 +2146,349 @@ def aggregate_cems_to_subplant(cems):
     cems = apply_dtypes(cems)
 
     return cems
+
+
+class AnomalyScreeningFirstStep:
```

Comment:
Since we are copying all of these functions directly from another repo, and there are a lot of them, let's move these to a separate module in OGE that is imported into data cleaning.

Reply:
Sounds good